docs/guide_slash_and_interactions.rst

```diff
@@ -0,0 +1,297 @@
+however let's say we want to limit this to voice channels only.
+We can do so by adjusting our type hint to :class:`discord.VoiceChannel` instead of :class:`discord.TextChannel`.
+
+.. code-block:: python
+
+    import discord
+
+    from redbot.core import commands, app_commands
+
+    class MyCog(commands.Cog):
+        def __init__(self, bot):
+            self.bot = bot
+
+        @app_commands.command()
+        @app_commands.describe(channel="The channel you want to use")
+        async def channel(self, interaction: discord.Interaction, channel: discord.VoiceChannel):
+            await interaction.response.send_message(f"Your channel is {channel.mention}", ephemeral=True)
+
+With integer and float arguments, we can also specify a minimum and maximum value.
+This can also be done to strings to set a minimum and maximum length.
+These limits will be reflected within Discord when the user is filling out the command.
+
+.. code-block:: python
+
+    import discord
+
+    from redbot.core import commands, app_commands
+
+    class MyCog(commands.Cog):
+        def __init__(self, bot):
+            self.bot = bot
+
+        @app_commands.command()
+        @app_commands.describe(number="The number you want to use, max 10")
+        async def number(self, interaction: discord.Interaction, number: app_commands.Range[int, None, 10]):
+            await interaction.response.send_message(f"Your number is {number}", ephemeral=True)
+
+See the `Discord.py documentation <https://discordpy.readthedocs.io/en/stable/interactions/api.html#range>`__ for more information on this.
+
+
+---------------
+Hybrid Commands
+---------------
+Hybrid commands are a way to bridge the gap between text commands and slash commands.
+These types of commands allow you to write a text and slash command simultaneously using the same function.
+This is useful for commands that you want to be able to use in both text and slash commands.
+
+.. note::
+    As with slash command groups, top level hybrid command groups **cannot** be invoked as a slash command. They can however be invoked as a text command.
+
+.. code-block:: python
+
+    from redbot.core import commands
+
+    class MyCog(commands.Cog):
+        def __init__(self, bot):
+            self.bot = bot
+
+        @commands.hybrid_command(name="cat")
+        async def cat(self, ctx: commands.Context):
+            await ctx.send("Meow")
+
+        @commands.hybrid_group(name="dog")
+        async def dog(self, ctx: commands.Context):
+            await ctx.send("Woof")
+            # As discussed above, top level hybrid command groups cannot be invoked as a slash command.
+            # Thus, this will not work as a slash command.
+
+        @dog.command(name="bark")
+        async def bark(self, ctx: commands.Context):
+            await ctx.send("Bark")
+
+After syncing your cog via the :ref:`Slash<core-command-slash>` command, you'll be able to use the commands as both a slash and text command.
```

Comment:
Yeah... this is a little late lol. The other examples don't point out syncing. Like I said above, add a brief explanation of the enable/sync process for the _first_ command, then assume they know it for the rest.